dynamic-demo-plugin/src/components/YAMLEditorPage.tsx

```diff
@@ -15,11 +15,11 @@ const EditorPage: React.FC = () => {
   const [data, setData] = React.useState(simpleYAML);
   const onSave = (content: string) => setData(yamlParser.load(content));
   return (
-    <Page>
+    <PageSection>
```

Comment:
YAML editor height is limited to only several lines
![Screenshot 2025-02-12 at 2 52 38 PM](https://github.com/user-attachments/assets/24499388-ff1b-41bf-9642-681de6eef815)

Reply:
Great catch.  Fixed.
<img width="1731" alt="Screenshot 2025-02-12 at 9 40 09 AM" src="https://github.com/user-attachments/assets/b206a54f-d986-48f8-a124-f404ad5cdcb7" />